infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -419,19 +405,23 @@ private void trimPrefix() {
         }
 
         // Close segments before deleting their corresponding log files
-        closeSegmentHandlers(endSegment);
+        segmentSupervisor.closeSegmentHandlers(endSegment);
 
+        deleteFiles(endSegment, "trimPrefix: ignoring file {}");
+
+        log.info("trimPrefix: completed, end segment {}", endSegment);
+    }
+
+    private void deleteFiles(long endSegment, String errorMsg) {
```

Comment:
Can make this `final long endSegment`

Reply:
there is no reason for that, `final` word is a boilerplate here